### PR TITLE
Heart failure modeling updates

### PR DESCRIPTION
**Pull request by SylLutze**

### CODE REVIEW

File: docs/source/gbd2019_models/causes/heart_failure/index.rst [hunk 1]
```diff
@@ -155,12 +142,63 @@ Vivarium Modeling Strategy
...
+     - 
+   * - 130-139 
+     - 1.5 (1.3, 1.73) 
+     - 
+   * - 140+ 
+     - 1.76 (1.43, 2.17) 
+     - 
+
+
+.. list-table:: Relative risk of heart failure for LDL-C
```
Comment (aflaxman) [inline docs/source/gbd2019_models/causes/heart_failure/index.rst:177]: Did Greg say today that we should not include a risk-outcome pair for LDL-C to HF?

**Review by aflaxman — APPROVED** [review]
Was the concept_model.rst intended to be in this PR?  Maybe it is a hold-over from a previous PR.

File: docs/source/gbd2019_models/causes/heart_failure/index.rst (hunk 1 above)
Comment (SylLutze) [reply docs/source/gbd2019_models/causes/heart_failure/index.rst]: Yeah - just updated this to reflect his latest email.

**Review by alibow — APPROVED** [review]
I just want to confirm that we won't be double-counting heart failure due to IHD with this modeling strategy and the modeling strategy for IHD in the same model - otherwise looks good to me

**Pull request merged**